substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -1406,27 +1406,17 @@ pub mod env {
 	/// Retrieve the code hash for a specified contract address.
 	/// See [`pallet_revive_uapi::HostFn::code_hash`].
 	#[api_version(0)]
-	fn code_hash(
-		&mut self,
-		memory: &mut M,
-		addr_ptr: u32,
-		out_ptr: u32,
-	) -> Result<ReturnErrorCode, TrapReason> {
+	fn code_hash(&mut self, memory: &mut M, addr_ptr: u32, out_ptr: u32) -> Result<(), TrapReason> {
```

Comment:
Nit: This and the uapi part are untested (i.e. there is no fixture)

Reply:
I can add one

Reply:
@xermicus done